modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMRoles.testcase

```diff
@@ -0,0 +1,50 @@
+@component-name = "OSB Site Initializer Partner Portal"
+definition {
+
+	var baseURL = PropsUtil.get("portal.url");
+
+	property custom.properties = "feature.flag.LPS-135430=true";
+	property osgi.modules.includes = "osb-site-initializer-partner-portal";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "OSB Site Initializer Partner Portal";
+
+	setUp {
+		task ("Set up instance and sign in") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+
+		task ("Create a new Partner Portal Site") {
+			PRMUtils.addPRMSite();
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Partner Portal");
+		}
+	}
+
+	@description = "Verify that new roles appear after create PRM site"
+	@priority = "5"
+	test CheckCreatedRoles {
+		task ("When: I go to Roles Page in Control Panel") {
+			Role.openRolesAdmin();
+
+			for (var searchTerm : list "Channel Account Manager,Channel General Manager,Channel Finance Manager,Channel Global Marketing Manager,Channel Regional Marketing Manager") {
+				Search.searchCP(searchTerm = "${searchTerm}");
+
+				LexiconEntry.viewEntryName(rowEntry = "${searchTerm}");
+			}
+		}
```

Comment:
Hi @heiditrindade, could you please split this task? You can add a new one describing that you are searching and seeing the PRM Roles.